docs/azure_container_apps.md

```diff
@@ -0,0 +1,33 @@
+# Deploying on Azure Container Apps
```

Comment:
Is this option compatible with all the features, notably private endpoints and user authentication? If not, can you add a note about that?

Reply:
it does look like some optional features are not supported from my understanding of the change

Reply:
I tested all optional features in https://github.com/Azure-Samples/azure-search-openai-demo/blob/main/docs/deploy_features.md except private endpoints(still in private preview for ACA) and Adding an OpenAI load balancer(a little bit complicated to follow).

All other features work, including user authentication and user document upload if I'm using my own Azure tenant (msft tenant has policy against it).

Reply:
`docs/azure_container_apps.md` is updated with a note saying `Private endpoints is still in private preview for Azure Conainer Apps and not supported for now.`

Reply:
Does the user login work? (https://github.com/Azure-Samples/azure-search-openai-demo/blob/main/docs/login_and_acl.md) For App Service, it works with built-in authentication, but maybe for Container Apps, it would work anyway thanks to our use of the MSAL SDK. The built-in authentication aspect is technically optional. Wondering if you tried out that feature.